DOC-2864 added two timestamp functions (#291)
tigergraph/gsql-docs: modules/ddl-and-loading/pages/functions/token/gsql_ts_to_epoch.adoc

= gsql_ts_to_epoch_seconds()

> Converts a timestamp in canonical string format to Unix epoch time, which is the number of seconds since `Jan 1st, 1970`.

Reviewer: ```suggestion
Converts a timestamp in canonical string format to Unix epoch time, which is the number of seconds since Jan 1, 1970 00:00 .
```

timestamp::
Required.
The timestamp to convert to Unix epoch time.
The timestamp parameter should be in one of the following formats:
* `"%Y-%m-%d %H:%M:%S"`
* `"%Y/%m/%d %H:%M:%S"`
* `"%Y-%m-%dT%H:%M:%S.000z"`
** Text after the dot `.` is ignored

== Return type
`UINT`

* Returns the number of seconds since `1970-01-01 00:00:00` for valid timestamps within the supported range.
* Throws an exception if the timestamp is invalid or before `1970-01-01 00:00:00`.

== Example

----
gsql_ts_to_epoch_seconds("2022-06-30 22-08-28") -> 1656652105
----

:table-caption!:
.Comparison of Timestamp Conversion Functions
[%autowidth]
|===

> |Function |Return type |Supported Time Range |Behavior |Description

Reviewer: ```suggestion
|Function |Return type |Supported Time Range |Response to Invalid/Unsupported Input|Description
```

> |Function |Return type |Supported Time Range |Behavior |Description

Reviewer: ```suggestion
|Function |Return type |Supported Time Range |Response to Invalid/Unsupported Datetimes |Description
```
